tutorials/vue/v-if-and-else.md

```diff
@@ -0,0 +1,50 @@
+To conditionally render something in vue, you must use `v-if` and `v-else` directive.
```

Comment:
Uppercase Vue. Also change the bit after the comma to "you should use the [`v-if` directive](https://masteringjs.io/tutorials/vue/v-if)". You don't _have_ to use `v-else`, or even `v-if` for that matter.